sdk/identity/identity/src/credentials/azureCliCredential.ts

```diff
@@ -149,7 +149,7 @@ export class AzureCliCredential implements TokenCredential {
           if (obj.stderr) {
             throw new CredentialUnavailableError(obj.stderr);
           }
-          throw e;
+          throw new CredentialUnavailableError(e);
```

Comment:
do we need to wrap this at both places?  Seems that changes in the outer catch block would handle any non-CredentialUnavailableError

Reply:
Fixed the double wrapping of Credential Unavailable Error happening.